### PR TITLE
[php-symfony] Set required PHP version ^7.1.3

### LINKED ISSUE #6081: [BUG][php-symfony] Composer installation fails under PHP 7.2
#### Bug Report Checklist

- [x] Have you provided a full/minimal spec to reproduce the issue?
- [x] Have you validated the input using an OpenAPI validator ([example](https://apidevtools.org/swagger-parser/online/))?
- [x] What's the version of OpenAPI Generator used?
- [x] Have you search for related issues/PRs?
- [x] What's the actual output vs expected output?
- [ ] [Optional] Bounty to sponsor the fix ([example](https://www.bountysource.com/issues/66123212-javascript-client-produces-a-wrong-object-for-a-string-enum-type-that-is-used-with-ref))

##### Description

During review of #6078 found following installation problem:
```console
$ composer install
Deprecation warning: Your package name GIT_USER_ID/GIT_REPO_ID is invalid, it should not contain uppercase characters. We suggest using git_user_id/git_repo_id instead. Make sure you fix this as Composer 2.0 will error.
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - friendsofphp/php-cs-fixer v1.13.3 requires php ^5.3.6 || >=7.0 <7.2 -> your PHP version (7.2.21) does not satisfy that requirement.
    - friendsofphp/php-cs-fixer v1.13.2 requires php ^5.3.6 || >=7.0 <7.2 -> your PHP version (7.2.21) does not satisfy that requirement.
    - friendsofphp/php-cs-fixer v1.13.1 requires php ^5.3.6 || >=7.0 <7.2 -> your PHP version (7.2.21) does not satisfy that requirement.
    - friendsofphp/php-cs-fixer v1.13.0 requires php ^5.3.6 || >=7.0 <7.2 -> your PHP version (7.2.21) does not satisfy that requirement.
    - friendsofphp/php-cs-fixer v1.12.4 requires php ^5.3.6 || >=7.0 <7.2 -> your PHP version (7.2.21) does not satisfy that requirement.
    - friendsofphp/php-cs-fixer v1.12.3 requires php ^5.3.6 || >=7.0 <7.2 -> your PHP version (7.2.21) does not satisfy that requirement.
    - friendsofphp/php-cs-fixer v1.12.2 requires php ^5.3.6 || >=7.0 <7.2 -> your PHP version (7.2.21) does not satisfy that requirement.
    - friendsofphp/php-cs-fixer v1.12.1 requires php ^5.3.6 || >=7.0 <7.2 -> your PHP version (7.2.21) does not satisfy that requirement.
    - friendsofphp/php-cs-fixer v1.12.0 requires php ^5.3.6 || >=7.0 <7.2 -> your PHP version (7.2.21) does not satisfy that requirement.
    - Installation request for friendsofphp/php-cs-fixer ~1.12 -> satisfiable by friendsofphp/php-cs-fixer[v1.12.0, v1.12.1, v1.12.2, v1.12.3, v1.12.4, v1.13.0, v1.13.1, v1.13.2, v1.13.3].
```

##### openapi-generator version

Current `master` branch.
Which is more important is tested PHP version:
```console
$ php -v
PHP 7.2.21 (cli) (built: Aug 11 2019 20:58:12) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.2.0, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.2.21, Copyright (c) 1999-2018, by Zend Technologies
    with Xdebug v2.7.2, Copyright (c) 2002-2019, by Derick Rethans
```

##### OpenAPI declaration file content or url

Build under samples directory.

##### Command line used for generation

Build under samples directory.

##### Steps to reproduce

```console
$ cd samples/server/petstore/php-symfony/SymfonyBundle-php
$ composer install
```

##### Suggest a fix

All dependencies should be specified as `*` or build should be tested under different PHP versions.

cc @jebentier @dkarlovi @mandrean @jfastnacht @ackintosh @renepardon 
cc PR's author @reznikartem

### LINKED ISSUE #6082: [BUG][php-symfony] Composer installation fails under PHP 5.4
#### Bug Report Checklist

- [x] Have you provided a full/minimal spec to reproduce the issue?
- [x] Have you validated the input using an OpenAPI validator ([example](https://apidevtools.org/swagger-parser/online/))?
- [x] What's the version of OpenAPI Generator used?
- [x] Have you search for related issues/PRs?
- [x] What's the actual output vs expected output?
- [ ] [Optional] Bounty to sponsor the fix ([example](https://www.bountysource.com/issues/66123212-javascript-client-produces-a-wrong-object-for-a-string-enum-type-that-is-used-with-ref))

##### Description

During review of #6078 found following installation problem:
```console
$ composer install
Deprecation warning: Your package name GIT_USER_ID/GIT_REPO_ID is invalid, it should not contain uppercase characters. We suggest using git_user_id/git_repo_id instead. Make sure you fix this as Composer 2.0 will error.
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - symfony/framework-bundle v4.4.7 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.4.6 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.4.5 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.4.4 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.4.3 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.4.2 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.4.1 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.4.0 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.3.9 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.3.8 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.3.7 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.3.6 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.3.5 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.3.4 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.3.3 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.3.2 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.3.11 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.3.10 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.3.1 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.3.0 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.2.9 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.2.8 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.2.7 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.2.6 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.2.5 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.2.4 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.2.3 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.2.2 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.2.12 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.2.11 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.2.10 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.2.1 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.2.0 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.1.9 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.1.8 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.1.7 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.1.6 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.1.5 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.1.4 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.1.3 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.1.2 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.1.12 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.1.11 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.1.10 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.1.1 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v4.1.0 requires php ^7.1.3 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.9 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.8 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.7 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.6 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.5 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.4 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.39 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.38 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.37 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.36 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.35 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.34 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.33 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.32 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.31 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.30 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.3 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.29 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.28 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.27 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.26 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.25 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.24 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.23 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.22 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.21 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.20 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.2 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.19 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.18 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.17 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.16 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.15 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.14 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.13 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.12 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.11 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.10 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.1 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.4.0 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.3.9 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.3.8 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.3.7 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.3.6 requires php >=5.5.9 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.3.5 requires php >=5.5.9 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.3.4 requires php >=5.5.9 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.3.3 requires php >=5.5.9 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.3.2 requires php >=5.5.9 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.3.18 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.3.17 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.3.16 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.3.15 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.3.14 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.3.13 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.3.12 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.3.11 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.3.10 requires php ^5.5.9|>=7.0.8 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.3.1 requires php >=5.5.9 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - symfony/framework-bundle v3.3.0 requires php >=5.5.9 -> your PHP version (7.2.21) overridden by "config.platform.php" version (5.4) does not satisfy that requirement.
    - Installation request for symfony/framework-bundle ^3.3|^4.1 -> satisfiable by symfony/framework-bundle[v3.3.0, v3.3.1, v3.3.10, v3.3.11, v3.3.12, v3.3.13, v3.3.14, v3.3.15, v3.3.16, v3.3.17, v3.3.18, v3.3.2, v3.3.3, v3.3.4, v3.3.5, v3.3.6, v3.3.7, v3.3.8, v3.3.9, v3.4.0, v3.4.1, v3.4.10, v3.4.11, v3.4.12, v3.4.13, v3.4.14, v3.4.15, v3.4.16, v3.4.17, v3.4.18, v3.4.19, v3.4.2, v3.4.20, v3.4.21, v3.4.22, v3.4.23, v3.4.24, v3.4.25, v3.4.26, v3.4.27, v3.4.28, v3.4.29, v3.4.3, v3.4.30, v3.4.31, v3.4.32, v3.4.33, v3.4.34, v3.4.35, v3.4.36, v3.4.37, v3.4.38, v3.4.39, v3.4.4, v3.4.5, v3.4.6, v3.4.7, v3.4.8, v3.4.9, v4.1.0, v4.1.1, v4.1.10, v4.1.11, v4.1.12, v4.1.2, v4.1.3, v4.1.4, v4.1.5, v4.1.6, v4.1.7, v4.1.8, v4.1.9, v4.2.0, v4.2.1, v4.2.10, v4.2.11, v4.2.12, v4.2.2, v4.2.3, v4.2.4, v4.2.5, v4.2.6, v4.2.7, v4.2.8, v4.2.9, v4.3.0, v4.3.1, v4.3.10, v4.3.11, v4.3.2, v4.3.3, v4.3.4, v4.3.5, v4.3.6, v4.3.7, v4.3.8, v4.3.9, v4.4.0, v4.4.1, v4.4.2, v4.4.3, v4.4.4, v4.4.5, v4.4.6, v4.4.7].
```

##### openapi-generator version

Current `master` branch.
Which is more important is tested PHP version in `composer.json`:
```json
{
...
    "config": {
        "platform": {
            "php": "5.4"
        }
    }
}
```

##### OpenAPI declaration file content or url

Build under samples directory.

##### Command line used for generation

Build under samples directory.

##### Steps to reproduce

```console
$ cd samples/server/petstore/php-symfony/SymfonyBundle-php
$ composer install
```

##### Related issues/PRs

Doesn't compiles with PHP 7.2 too #6081 

##### Suggest a fix

#1246 suggested to use minimal PHP version 7.x long time ago. Need to upgrade composer template and test installation with emulated PHP 7.0/7.1/7.2. Emulation can be done with [platform Composer option](https://getcomposer.org/doc/06-config.md#platform).

cc @jebentier @dkarlovi @mandrean @jfastnacht @ackintosh @renepardon 
cc PR's author @reznikartem

### PR DESCRIPTION
**Pull request by ybelenko**
Changes:
* Minimal PHP version set to ^7.3.1
* `symfony/framework-bundle` version bumped to ^4.4.8
* `phpunit` version bumped to ^7.0
* `php-cs-fixer` version bumped to ^2.16.3
* `php_codesniffer ` has been removed
* `php-coveralls` has been removed
* Updated Travis CI config
* Added Basic Coveralls config
* `composer.lock` excluded from version control

Build has been tested under PHP 7.2.21(local machine) and PHP 7.1.3(`{"config": {"platform": {"php": "7.1.3"}}}`).

Related issue about end of PHP 5.x #1246 
Fixes #6081, fixes #6082 

cc @jebentier @dkarlovi @mandrean @jfastnacht @ackintosh @renepardon
cc @reznikartem as last Symfony contributor

### PR checklist

- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [x] If contributing template-only or documentation-only changes which will change sample output, [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) before.
- [x] Run the shell script(s) under `./bin/` (or Windows batch scripts under`.\bin\windows`) to update Petstore samples related to your fix. This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit, and these must match the expectations made by your contribution. You only need to run `./bin/{LANG}-petstore.sh`, `./bin/openapi3/{LANG}-petstore.sh` if updating the code or mustache templates for a language (`{LANG}`) (e.g. php, ruby, python, etc).
- [x] File the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`, `4.3.x`, `5.0.x`. Default: `master`.
- [x] Copy the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.

### CODE REVIEW

File: modules/openapi-generator/src/main/resources/php-symfony/.coveralls.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# same as Ruby lib
```
Comment (wing328) [inline modules/openapi-generator/src/main/resources/php-symfony/.coveralls.yml:1]: Shall it be PHP lib? I don't how Ruby is related in this PR.

Comment (ybelenko) [reply modules/openapi-generator/src/main/resources/php-symfony/.coveralls.yml:1]: Yeah, looks weird, but it's just a comment. I've blindly copied it from official [.coveralls.yml example configuration](https://github.com/php-coveralls/php-coveralls#coverallsyml) few years ago. It works, trust me. :smile:

Comment (ybelenko) [reply modules/openapi-generator/src/main/resources/php-symfony/.coveralls.yml:1]: I can remove that comment to avoid misunderstanding.

**Comment by FlaverSaver** [comment]
I think we should take this chance and drop support for the outdated PHP 5 version.
It would also be great if we could support Symfony 5 in addition to Symfony 4. At first look, the main work is to upgrade the jms/serializer package from version 1 to version 3. To do this, the class StrictJsonDeserializationVisitor has to be adapted. See: https://github.com/schmittjoh/serializer/blob/master/UPGRADING.md#from-1x-to-300

File: modules/openapi-generator/src/main/resources/php-symfony/.coveralls.yml (hunk 1 above)
Comment (wing328) [reply modules/openapi-generator/src/main/resources/php-symfony/.coveralls.yml:1]: Yes please as someone else may also find this confusing.

**Pull request merged**